src/containers/JoinUs.tsx

```diff
@@ -0,0 +1,71 @@
+'use client';
+
+import React, { useState } from 'react';
+import Image from 'next/image';
+
+import CInput from '../components/Cinput';
+import CButton from '../components/Cbutton';
+import { ExternalPages } from '../constants/externalPages';
+
+import fetch from '../utils/request';
+
+import JoinDancing from 'public/images/joinDancing.png';
+
+const JoinUs = () => {
+  const [value, setValue] = useState('');
+
+  const onChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setValue(event.target.value);
+  };
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    fetch(ExternalPages.FLUXITY_API + '/subscribe', {
+      method: 'POST',
+      body: JSON.stringify({
+        email: value,
+      }),
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
```

Comment:
use `.then` and `.catch` to check the response. If it fails for any reasons, show a toast to the user. If it succeeds, show a toast to the user.